pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/DSMtasksSubscription.java

```diff
@@ -120,7 +120,7 @@ public static void doESExport(Map<String, String> attributesMap, String data) {
         }
     }
 
-    public static void migrateToES(ExportToES.ExportPayload exportPayload) {
+    public static boolean migrateToES(ExportToES.ExportPayload exportPayload) {
```

Comment:
When would this be expected to return false?

Reply:
No I changed it so the functionality can be used synchronously in test, and tests wait for a return value from this method to make sure export is completed.